source/bayes_simulation.Rmd

```diff
@@ -45,112 +44,268 @@ correctly identifies 80 percent as "OK" but says that 20 percent are
+to a problem should be. Thereafter, we can look into different—sometimes more elegant or accurate—ways of modeling and solving it.
 
-`           IF a between 28 86         `
+Let's examine the following tree diagram of our car selection:
 
-If it's between "28" and "86" (those that say "good")
+![](diagrams/car-tree.png)
 
-`           SCORE a z         `
+We see that there are two paths, highlighted, that results in a car
+being approved by a mechanic.  Either a car can be working, and
+correctly identified as such by a mechanic; or the car can be broken,
+while the mechanic mistakenly determines it to be working. Our
+question only pertains to these two paths, so we do not need to
+study the rest of the tree.
 
-Score this number
+We want to know, of all the cars approved by a mechanic, how many are faulty:
 
-`           END         `
+$$ \frac{\mathrm{cars_{\mathrm{faulty}}}}{\mathrm{cars}_{\mathrm{approved}}} $$
 
-End the IF condition
+We saw that there are two paths in the tree that lead to a mechanic
+approving a car, and we can calculate the fraction of cars that end up so:
 
-`           END         `
+1. Of all cars, 30% are faulty.  Of these, 10% are approved by a mechanic.  That is,
+   30% * 10% = 3% of all cars.
+2. Of all cars, 70% work.  Of these, 80% are approved by a mechanic.  That is,
+   70% * 80% = 56% of all cars.
 
-End REPEAT loop
+The percentage of faulty cars, out of approved cars, becomes:
 
-`           COUNT z between 28 30 k         `
+$$ 3% / (56% + 3%) = 5.08%. $$
 
-How many of the SCORED numbers were between "28\--30" (faulty, "OK")
+Notation-wise, it is a bit easier to calculate these sums using fractions rather than percentages:
 
-`           SIZE z s         `
+1. Faulty cars approved by a mechanic: 0.3 * 0.1 = 0.03
+2. Working cars approved by a mechanic: 0.7 * 0.8 = 0.56
 
-How many numbers were scored
+Fraction of faulty cars out of approved cars: 0.03 / (0.03 + 0.56) = 0.0508
 
-`           DIVIDE k s kk         `
+We see that every time the tree branches, it filters the cars: some go to one branch, the rest to another.  In our code, we used the AND (`&`) operator to find the intersection between faulty AND approved cars, i.e., to filter out from all faulty cars only the cars that were ALSO approved.
 
-What proportion were faulty, "OK"
+### Probability interpretation
 
-`           PRINT kk         `
+**Probability from proportion**
 
-Print result
+In these examples, we often calculate proportions.
+In the given simulation:
+How many cars are approved by a mechanic?  59/100.
+How many of those 59 were faulty? 3/59.
+We also count how commonly events occur: "it rained 4 out of the 10 days".
 
-Result kk = 0.039
+An extension of this idea is to *predict* the probability of an event occurring, based on what we had seen in the past.
+We can say "out of 100 observations, condition X occurred 20 times; therefore, I  estimate that the probability of X occurring, $P(X)$, is 20/100".
```

Comment:
Can you think of a good example, from which to generalize directly?  Even, "out of 100 days, there was some rain on 20 days.  More generally, we can say ...".